README.md

```diff
@@ -1 +1,181 @@
-# planship-vue
\ No newline at end of file
+# planship-vue
+
+Welcome to `@planship/vue`, a [Vue plugin](https://vuejs.org/guide/reusability/plugins) that enables entitlements, metering, plan packaging, and customer/subscription management in your Vue app powered by [Planship](https://planship.io). This plugin is built on top of the [@planship/fetch](https://github.com/planship/planship-js/tree/master/packages/fetch) JavaScript library, and it uses the [Vue provide/inject mechanism](https://vuejs.org/guide/components/provide-inject).
+
+To use the plugin in [Nuxt](https://nuxt.com/) applications, including [universal](https://nuxt.com/docs/guide/concepts/rendering#universal-rendering) (server-side + client-side) rendering apps, use the [`@planship/nuxt`]('https://github.com/planship/planship-nuxt') module instead.
+
+## How does it work?
+
+The plugin, once initialized, exposes the Planship API client and customer entitlements to all pages and components of the application.
+
+#### `usePlanshipCustomer`
+
+Returns an instance of the [Planship Customer API client](https://github.com/planship/planship-js/blob/master/packages/fetch/docs/interfaces/PlanshipCustomerApi.md) and customer entitlements.
+
+```js
+import { usePlanshipCustomer } from '@planship/vue'
+
+const { planshipCustomerApiClient, entitlements } = await usePlanshipCustomer('<CUSTOMER_ID>')
+```
+
+#### `usePlanship`
+
+Returns an intance of the [Planship API client](https://github.com/planship/planship-js/blob/master/packages/fetch/docs/interfaces/PlanshipApi.md).
+
+```js
+import { usePlanship } from '@planship/vue'
+
+const { planshipApiClient } = usePlanship()
+```
+
+## Getting started
+
+To use the plugin in [Nuxt](https://nuxt.com/) applications, including [universal](https://nuxt.com/docs/guide/concepts/rendering#universal-rendering) (server-side + client-side) rendering apps, use the [`@planship/nuxt`]('https://github.com/planship/planship-nuxt') module instead.
```

Comment:
Same suggestion as comment at the top of the readme.

Reply:
I dropped the part about universal rendering, simplified it to this:
If you're building [Nuxt](https://nuxt.com/) applications, use the [`@planship/nuxt`]('https://github.com/planship/planship-nuxt') module instead.